pkg/status/status.go

```diff
@@ -0,0 +1,70 @@
+package status
+
+import (
+	"context"
+	"time"
+
+	metallbv1alpha1 "github.com/metallb/metallb-operator/api/v1alpha1"
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	ConditionAvailable   string = "Available"
```

Comment:
No need to specify the type with const

Reply:
inline the value? It's used in several places, so I would prefer to have a var for this

Reply:
No I mean, you can omit `string`